app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java

```diff
@@ -394,13 +398,21 @@ public void onBindViewHolder(Presenter.ViewHolder viewHolder, Object item) {
             holder.mCardView.setPlayingIndicator(true);
         } else {
             holder.mCardView.setPlayingIndicator(false);
-            Drawable badge = rowItem.getBadgeImage();
-            if (badge != null) {
-                ((ViewHolder) viewHolder).mCardView.setBadgeImage(badge);
+
+            if (get(UserPreferences.class).get(UserPreferences.Companion.getDefaultRatingProvider()) == RatingProvider.RATING_TOMATOES) {
```

Comment:
I think the code will be more readable if the result of the `get(UserPreferences.class).get(UserPreferences.Companion.getDefaultRatingProvider())` part is moved to a variable before the if statement.